CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ All notable changes to this project will be documented in this file.
   * #### Added
   * #### Changed
   * #### Removed
+    * Removed init_with_topology. Clients should use niswitch.Session.init(). See #660.
```

Comment:
There is no niswitch.Session.init()
There is `niswitch.Session.__init__()`. I think Markdown rendering screwed things up. You can change that to

> Clients should use niswitch.Session() constructor.